### PR TITLE
Refactor collectMetrics command

### PR DESCRIPTION
**Pull request by ssz1997**
1. We added more information to the response JSON string. Make the whole thing a JSON string for easier parsing.
2. Refactor the get metrics part for reusing.

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/bundler/command/CollectMetricsCommand.java [hunk 1]
```diff
@@ -122,62 +158,46 @@ private void masterMetrics(StringWriter outputBuffer, int i) throws IOException
       // Do not break the loop since the HTTP failure can be due to many reasons
       // Return the error message instead
       LOG.error("Failed to get Alluxio master metrics from URL {}. Exception: ", url, e);
-      metricsResponse =  String.format("Url: %s%nError: %s", url, e.getMessage());
+      metricsResponse =  String.format("{Url: \"%s\",%n\"Error\": %s}", url, e.getMessage());
```
Comment (jiacheliu3) [inline shell/src/main/java/alluxio/cli/bundler/command/CollectMetricsCommand.java:161]: im fine with this manual wrapping for now. An alternative is to wrap it with an object like
```
class CollectMetricsResponse {
  String url;
  String error;
  String body;
}
```
Then use jackson ObjectMapper to convert to json. Up to you :)

**Review by jiacheliu3 — APPROVED** [review]
LGTM, could you attach one sample output file/content to the PR? Thanks!

**Comment by ssz1997** [comment]
[collectMetrics.txt](https://github.com/Alluxio/alluxio/files/11013181/collectMetrics.txt)
By default we take 3 snapshots of the metrics. Running Alluxio locally w/ 1 master and 1 worker and this is the file containing all records under `<ALLUXIO_WORK_DIR>/.collectInfo/collectMetrics/collectMetrics.txt`

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Pull request merged**